TbsCore/Core/WebBrowserInfo.cs

```diff
@@ -235,7 +235,7 @@ public string GetBearerToken()
         {
             CheckChromeOpen();
             IJavaScriptExecutor js = acc.Wb.Driver;
-            return (string)js.ExecuteScript("for(let field in Travian) { if (Travian[field].length == 32) return Travian[field]; }");
+            return (string)js.ExecuteScript("return Travian.abrogatedReanimatingAftereffects");

```

Comment:
I'm fairly sure Travian changes this string every session, and `abrogatedReanimatingAftereffects` is 32 characters so it should find it as well